DSC/README.md

```diff
@@ -25,7 +25,12 @@ Here're all the supported public configuration parameters:
 
 * `FileUri`: (optional, string) the uri of the MOF file/Meta MOF file/custom resource ZIP file.
 * `ResourceName`: (optional, string) the name of the custom resource module
-* `Mode`: (optional, string) the functional mode, valid values: Push, Pull, Install, Remove, Register. If not specified, it's considered as Push mode by default.
+* `Operation`: (optional, string) the functional operation, valid values: Register, Push, Pull, Install, Remove. If not specified, it's considered as Push Operation by default.
+* `NodeConfigurationName`: (optional, string) the name of the configuration to apply.
+* `RefreshFrequencyMins`: (optional, int) pecifies how often (in minutes) DSC attempts to obtain the configuration from the pull server. 
+       If configuration on the pull server differs from the current one on the target node, it is copied to the pending store and applied.
+* `ConfigurationMode`: (optional, string) Specifies how DSC should apply the configuration. Valid values are: ApplyOnly, ApplyAndMonitor, ApplyAndAutoCorrect.
```

Comment:
I copied this stmt from DSC for linux README page. Current statement helps customer to understand better.